### DISCUSSION TITLE
Coral USB status LED?

### DISCUSSION THREAD

**Discussion by EM-Tom** [blakeblackshear/frigate#3554] [Ask A Question] [answered]
Hey all, 

Running 11 RC1 with great success at present.

I'm sorry this isn't massively related to Frigate, but this is a community of people who have Corals and I will be using mine for Frigate. I was wondering if anyone could confirm if a USB Coral illuminates an LED when it's powered on? Docs suggest it should. I got one delivered today which I fear is DOA, no OS can see the USB device and before I raise a case with the vendor wanted to make sure I am right. 

I've googled and tried to look at all sorts of pics without success.

**Comment by NickM-27** [comment, marked as answer]
The USB coral has a white LED, in my experience it flashes when being in use (like when frigate is running). Not sure if it is constant. No OS sees the USB device at all, even as a generic USB device?

**Reply by EM-Tom** [reply]
Thanks for your super fast reply. No LED lifghts at all, I can see the diode. I'll return DOA. 

Can't see anything in /dev on Ubuntu or Mac  (Or ESXi) 
Can't see anything at all in Device Manager on Windows
